application/models/SurveyActivator.php

```diff
@@ -417,7 +417,8 @@ protected function createResponseTableKeys()
      */
     protected function createTimingsTable()
     {
-        if ($this->survey->isSaveTimings) {
+        $prow = $this->survey->find('sid = :sid', array(':sid' => $this->survey->sid));
```

Comment:
please add a comment here why you had to implement this construct

Reply:
Added a comment